addons/l10n_sg/data/l10n_sg_chart_data.xml

```diff
@@ -316,47 +316,47 @@
         </record>
 
         <record model="account.account.template" id="account_account_738">
-            <field name="name">Purchase Tax Account 8% TX-N33</field>
+            <field name="name">Purchase Tax Account 9% TX-N33</field>
```

Comment:
The only uncertainty I have with this pr is that renaming.
In 16.2+, I think the update script will also update the account name (not 100% sure) but I think it should be checked too.
Because in this case, it would not be very consistent until 2024 right.